api/v1alpha1/circuitbreaker_types.go

```diff
@@ -46,4 +46,20 @@ type CircuitBreaker struct {
 	// +kubebuilder:validation:Maximum=4294967295
 	// +optional
 	MaxRequestsPerConnection *int64 `json:"maxRequestsPerConnection,omitempty"`
+
+	// PerHost defines per-host Circuit Breakers
+	//
+	// +optional
+	PerHost *PerHostCircuitBreakers `json:"perHost,omitempty"`
+}
+
```

Comment:
can we move all the existing thresholds (MaxConnections, Max ...) into 1 struct, embed that in the top level and reuse it here ?

Reply:
ignore my comment, only per host max connections is supported so this is fine